Detectors/Base/include/DetectorsBase/DetID.h

```diff
@@ -0,0 +1,101 @@
+/// @copyright
+/// © Copyright 2014 Copyright Holders of the ALICE O2 collaboration.
+/// See https://aliceinfo.cern.ch/AliceO2 for details on the Copyright holders.
+/// This software is distributed under the terms of the
+/// GNU General Public License version 3 (GPL Version 3).
+///
+/// License text in a separate file.
+///
+/// In applying this license, CERN does not waive the privileges and immunities
+/// granted to it by virtue of its status as an Intergovernmental Organization
+/// or submit itself to any jurisdiction.
+
+/// @brief ALICE detectors ID's, names, masks
+///
+/// @author Ruben Shahoyan, ruben.shahoyan@cern.ch
+
+/*! 
+  Example of class usage:
+  using namespace o2::base;
+  DetID det[3] = {DetID(DetID::ITS), DetID(DetID::TPC), DetID(DetID::TRD)};
+  int mskTot = 0;
+  for (int i=0;i<3;i++) {
+    printf("detID: %2d %10s 0x%x\n",det[i].getID(),det[i].getName(),det[i].getMask());
+    mskTot |= det[i].getMask();
+  }
+  printf("joint mask: 0x%x\n",mskTot);
+ */
+
+
+
+#ifndef O2_BASE_DETID_
+#define O2_BASE_DETID_
+
+#include <cstdint>
+
+namespace o2 {
+namespace Base {
```

Comment:
not sure now but I think the style guide says lowercase for all namespaces (which we can change later for Base)